### PR TITLE
enhancement/1123 - Make runway mandatory in "taxi" command

### LINKED ISSUE #1123: Make runway mandatory in "taxi" command

### PR DESCRIPTION
**Pull request by felixscheffer**
Resolves #1123

### CODE REVIEW

File: src/assets/scripts/client/aircraft/AircraftCommander.js [hunk 1]
```diff
@@ -559,21 +559,24 @@ export default class AircraftCommander {
...
      *
-     * If no runway is specified, the aircraft will taxi to their expected departure runway, stored
-     * in Fms.departureRunwayModel. And if a runway is requested but doesn't exist, an error is returned.
+     * If a runway is requested but doesn't exist, an error is returned.
      *
      * @for AircraftCommander
      * @method runTaxi
      * @param {AircraftModel} aircraftModel
-     * @param {string} data
+     * @param {array} data
```
Comment (n8rzz) [inline src/assets/scripts/client/aircraft/AircraftCommander.js]:
```suggestion
     * @param {array<string>} data
```

File: src/assets/scripts/client/aircraft/AircraftCommander.js [hunk 2]
```diff
@@ -559,21 +559,24 @@ export default class AircraftCommander {
...
      * @return {array} [success of operation, readback]
      */
     runTaxi(aircraftModel, data) {
         const airportModel = AirportController.airport_get();
         const requestedRunwayName = data[0];
 
         if (!requestedRunwayName) {
-            return aircraftModel.taxiToRunway(airportModel.departureRunwayModel);
+            const readback = {};
+            readback.log = 'we did not understood the runway. say again';
```
Comment (n8rzz) [inline src/assets/scripts/client/aircraft/AircraftCommander.js]:
```suggestion
            readback.log = 'we did not understand the runway. say again';
```

File: src/assets/scripts/client/aircraft/AircraftCommander.js [hunk 3]
```diff
@@ -559,21 +559,24 @@ export default class AircraftCommander {
...
+            readback.say = 'we did not understood the runway. say again';
```
Comment (n8rzz) [inline src/assets/scripts/client/aircraft/AircraftCommander.js]:
```suggestion
            readback.say = 'we did not understand the runway. say again';
```

**Comment by felixscheffer** [comment]
Your wish is my command :smile:

**Pull request merged**